package io.github.absdf15.openai.exception



internal data class ErrorInfoWrapper(
    val error: ErrorInfo
)

